includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -825,6 +825,9 @@ public function needs_https_setup() {
 	 * @return bool Whether the gateway is enabled and ready to accept payments.
 	 */
 	public function is_available() {
+		if ( ! WC_Payments::get_gateway()->is_enabled() ) {
```

Comment:
This makes sense, but I still wonder why it wasn't part of the codebase prior 😅 I have been searching past changes, but couldn't find an answer in the ancient scriptures.